### ISSUE TITLE
Unable to run latest qubes-thunderbird add-on with thunderbird 78.8

### ISSUE THREAD

**Issue by one7two99** [QubesOS/qubes-issues#6484]
**Qubes OS version**
Qubes Release 4.0 (R4.0)

**Affected component(s) or functionality**
Unable to get a working combination of thunderbird and the qubes-thunderbird addon to be able to launch attachments.

**Brief summary**
tried to install the suggested testing repositories in the debian-10 and also a fedora-33 template.
alsdo downloaded the .deb package from https://ftp.qubes-os.org/repo/deb/r4.1/vm/pool/main/q/qubes-thunderbird/ and installed qubes-thunderbird_2.0.5-1+deb10u1_amd64.deb 
Thunderbird is Version 78.8.0 but it doesn't work with the qubes-thunderbird plugin version 2.0.5

**How Reproducible**
Install recent version of thunderbird 78.8 from the repositories.
install latest version of qubes-thunderbird add-on 2.0.5.
launch thunderbird -> qubes-thunderbird plugin will not work with this version.

**Expected behavior**
beeing able to use qubes-thunderbird add-on with thunderbird

**Actual behavior**
qubes-thunderbird add-on can't be used / is disabled.

**Solutions you've tried**
tried to install the latest versions from the testing repositores and also a manual dpkg -i *.deb install.

**Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**
https://github.com/QubesOS/qubes-issues/issues/5318

**Comment by DemiMarie** [comment]
This is a duplicate of #5318.  In short: a version that works with Thunderbird 78+ _will_ happen at some point, but it will be essentially a complete rewrite and is currently blocked on getting some needed APIs from the Thunderbird developers.  So don’t hold your breath.  In the meantime, R4.1 may provide a way to tell an application to open everything in a DispVM, which might provide a partial workaround.

**Comment by andrewdavidwong** [comment]
Duplicate of #5861

**Comment by andrewdavidwong** [comment]
This appears to be a [duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) of an existing issue. If so, please comment on the appropriate existing issue instead. If you believe this is not really a duplicate, please leave a comment briefly explaining why. We'll be happy to take another look and, if appropriate, reopen this issue. Thank you.

**Comment by one7two99** [comment]
but afaik there is no release date for qubes 4.1 yet.
as a workarround I have been able to reconfigure the AppVM, so that a click on an URL in thunderbird will launch a disposable vm by tweaking the links.desktop file in the appvm and replacing links with open-in-dispvm

[user@my-bizmail ~]$  cat /usr/share/applications/links.desktop
[Desktop Entry]
Version=1.0
Encoding=UTF-8
Name=Links
Comment=Simple Web Browser
Exec=qvm-open-in-dvm %u
Icon=links
Categories=Network;WebBrowser;
Type=Application
MimeType=text/html;application/xhtml+xml;x-scheme-handler/http;x-scheme-handler/https;
X-Desktop-File-Install-Version=0.24

- one7two99
